gradle/lib/dependabot/gradle/language.rb

```diff
@@ -0,0 +1,24 @@
+# typed: strong
+# frozen_string_literal: true
+
+require "sorbet-runtime"
+require "dependabot/ecosystem"
+require "dependabot/gradle/version"
+
+module Dependabot
+  module Gradle
+    LANGUAGE = "jvm_languages"
```

Comment:
Since Gradle can build [non-JVM projects](https://docs.gradle.org/current/userguide/userguide.html#supported_languages_and_frameworks) like C++ and JavaScript, is `jvm_languages` is the right value here?

Reply:
In that case, perhaps, I should just use unknown / not available

Reply:
Just FYI

To identify the programming language from a Gradle build file, you can look for specific plugins and configurations that are commonly associated with certain languages. Here are some key indicators:  

1. Java: Look for the java plugin.  
```plugins {
    id 'java'
}```
2. Kotlin: Look for the kotlin plugin.  
```plugins {
    id 'org.jetbrains.kotlin.jvm' version '1.5.21'
}```
3. Groovy: Look for the groovy plugin.  
```plugins {
    id 'groovy'
}```
4. Scala: Look for the scala plugin.  
```plugins {
    id 'scala'
}```
5. Android: Look for the com.android.application or com.android.library plugins.  
```plugins {
    id 'com.android.application'
}```
By examining the plugins section of the build.gradle file, you can determine the primary programming language used in the project.

And for 
6. C++
```
plugins {
    id 'cpp'
}
```
7. Javacript
```
plugins {
    id 'com.github.node-gradle.node' version '3.0.1'
}
```

If this information is not there then we can say unknown. This is my reading, Unblocking comment

Reply:
@thavaahariharangit can you provide a doc for this, it might be useful for @amazimbe

Reply:
If there is no plugins.id in the build.gradle file, you can look for other indicators to identify the programming language. Here are some additional clues:  

1. Java: Look for apply plugin: 'java' or dependencies like implementation 'org.springframework.boot:spring-boot-starter'.
2. Kotlin: Look for apply plugin: 'kotlin' or dependencies like implementation "org.jetbrains.kotlin:kotlin-stdlib".
3. Groovy: Look for apply plugin: 'groovy' or dependencies like implementation 'org.codehaus.groovy:groovy-all'.
4. Scala: Look for apply plugin: 'scala' or dependencies like implementation 'org.scala-lang:scala-library'.
5. Android: Look for apply plugin: 'com.android.application' or apply plugin: 'com.android.library'.

By examining these elements, you can determine the primary programming language used in the project.

#### JavaScript
Look for:  
apply plugin: 'node'
apply plugin: 'com.github.node-gradle.node'
Dependencies like implementation 'org.nodejs:node' or implementation 'com.github.node-gradle:node'
#### C++
Look for:  
apply plugin: 'cpp'
Dependencies like implementation 'org.gradle.cpp:cpp-library'
By examining these elements, you can determine if the project is using JavaScript or C++.

Reply:
I'll merge this but will create another PR if I manage to do the parsing consistently and reliably.